blog-service/2024-01-11-search.md

```diff
@@ -0,0 +1,20 @@
+---
+title: New Visualizations (Search) 
+image: https://www.sumologic.com/img/logo.svg
+keywords:
+  - log_level
+  - search
+  - log search
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+We're excited to announce new visualizations for Log Search queries with which you can have a unified experience with the charts across the Sumo Logic platforms. 
+
+Click the **New Visualizations** button to enable new visualizations for your log search.
```

Comment:
We should also include information about to updating the user preference when user toggle between new to old charts phase 1

Reply:
Content has been updated.